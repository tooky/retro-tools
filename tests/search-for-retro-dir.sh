retro_date=1997-06-05 retro add
mkdir subdir
cd subdir
retro_date=1997-06-06 retro add
cd ..
ls doc/retro/*/*/*
