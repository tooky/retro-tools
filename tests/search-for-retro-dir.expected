retro_date=1997-06-05 retro add
doc/retro/1997/06/05.md
mkdir subdir
cd subdir
retro_date=1997-06-06 retro add
../doc/retro/1997/06/06.md
cd ..
ls doc/retro/*/*/*
doc/retro/1997/06/05.md
doc/retro/1997/06/06.md
